config/jobs/kubernetes-sigs/jobset/jobset-periodics-release-0-4.yaml

```diff
@@ -0,0 +1,193 @@
+periodics:
+  - interval: 12h
+    name: periodic-jobset-test-unit-release-0-4
+    cluster: eks-prow-build-cluster
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: jobset
+        base_ref: release-0-4
+        path_alias: sigs.k8s.io/jobset
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: periodic-jobset-test-unit-release-0-4
+      testgrid-num-failures-to-alert: '1'
+      description: "Run periodic jobset unit tests"
+      testgrid-num-columns-recent: '30'
+    labels:
+      preset-dind-enabled: "true"
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240409-13cd3acf7e-master
+        env:
+        - name: GO_TEST_FLAGS
+          value: "-race -count 3"
+        command:
+        # generic runner script, handles DIND, bazelrc for caching, etc.
+        - runner.sh
+        args:
+        - make
+        - test
+        securityContext:
+          privileged: true
+        resources:
+          limits:
+            cpu: 3
+            memory: 10Gi
+          requests:
+            cpu: 3
+            memory: 10Gi
+  - interval: 12h
+    name: periodic-jobset-test-integration-release-0-4
+    cluster: eks-prow-build-cluster
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: jobset
+        base_ref: release-0-4
+        path_alias: sigs.k8s.io/jobset
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: periodic-jobset-test-integration-release-0-4
+      testgrid-num-failures-to-alert: '1'
+      description: "Run periodic jobset integration tests"
+      testgrid-num-columns-recent: '30'
+    labels:
+      preset-dind-enabled: "true"
+    decorate: true
+    spec:
+      containers:
+      - image: public.ecr.aws/docker/library/golang:1.22
+        command:
+        - make
+        args:
+        - test-integration
+        resources:
+          limits:
+            cpu: 3
+            memory: 10Gi
```

Comment:
10Gi seems like a lot, is this what Kueue uses? I don't think we need that much, see jobset controller manager resource requests/limits [here](https://github.com/kubernetes-sigs/jobset/blob/235745c5ed1bd70f54f36aa8782122e3326a04b6/config/components/manager/manager.yaml#L94-L100)

Reply:
Yes, kueue using 10Gib too , see [here](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-5.yaml#L95)

And the resources described here is not for the jobset controller manager, its for this container [here](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-5.yaml#L76)

Reply:
I see, ok let's stick with 3 for now and see how that performs, we can bump it if necessary.

Reply:
Sure 👍